tools/ansible/tasks/wsu/main.yaml

```diff
@@ -154,6 +161,10 @@
       shell: "oc label node {{ node_name.stdout_lines[0] }} node-role.kubernetes.io/worker="
       when: node_name.stdout_lines | length == 1 and checklabels.rc == 1
 
+    - name: Ensure the hybrid overlay is not running
+      win_shell: "Stop-Process -Name \"hybrid-overlay\""
+      ignore_errors: yes
```

Comment:
So what if the process does not stop in the case it was running before?

Reply:
Good call. I'll change that